cea/globalvar.py

```diff
@@ -77,7 +77,7 @@ def __init__(self):
         # Commands for the evolutionary algorithm
 
 
-        self.initialInd = 2  # number of initial individuals
+        self.initialInd = 5  # number of initial individuals
```

Comment:
we should move these to the configuration file. just keep that in mind. Also, pep8 all the variable names (but slowly).